src/main/java/com/thecookiezen/presentation/NodeController.java

```diff
@@ -0,0 +1,53 @@
+package com.thecookiezen.presentation;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.model.Container;
+import com.github.dockerjava.api.model.Info;
+import com.thecookiezen.bussiness.cluster.boundary.ClusterRepository;
+import com.thecookiezen.bussiness.cluster.entity.Cluster;
+import com.thecookiezen.bussiness.cluster.entity.DockerHost;
+import com.thecookiezen.bussiness.containers.boundary.ContainerFetcher;
+import lombok.AllArgsConstructor;
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.ws.rs.NotFoundException;
+import java.util.Collection;
+import java.util.Optional;
+
+@Controller
+@Log4j
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class NodeController {
+
+    ContainerFetcher containerFetcher;
+
+    ClusterRepository clusterRepository;
+
+    @RequestMapping("/clusters/node")
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2\.